### PR TITLE
Increase avatar image quality and size

### PR DESCRIPTION
**Pull request by benalleng**
builds on #681

### CODE REVIEW

File: content/lessons/chapter-3/coop-3.tsx [hunk 1]
```diff
@@ -196,6 +197,7 @@ export default function Coop3({ lang }) {
               key={i}
               username={profile.username}
               avatar={profile.avatar}
+              avatarSize={isSmallScreen ? 50 : 100}
```
Comment (0tuedon) [inline content/lessons/chapter-3/coop-3.tsx:200]: shouldn't the size be a 75? from the previous class i have seen min-w-[75px]

Comment (benalleng) [reply content/lessons/chapter-3/coop-3.tsx:200]: This is on a challenge page the 75px icons are on intro pages

image

vs

image

Comment (0tuedon) [reply content/lessons/chapter-3/coop-3.tsx:200]: oh yeah right. i remember that

**Review by 0tuedon — APPROVED** [review]
LGTM!

**Pull request merged**